perception/lidar_centerpoint/lib/postprocess/postprocess_kernel.cu

```diff
@@ -116,6 +117,9 @@ cudaError_t PostProcessCUDA::generateDetectedBoxes3D_launch(
   const float * out_rot, const float * out_vel, std::vector<Box3D> & det_boxes3d,
   cudaStream_t stream)
 {
+  thrust::device_vector<float> yaw_norm_thresholds(
+    config_.yaw_norm_thresholds_.begin(), config_.yaw_norm_thresholds_.end());
```

Comment:
I think it's better to declare `yaw_norm_thresholds` as as a class member since memcopy at once is enough. And in this package I use `_d` as a notation meaning the device_vector / device_pointer.

Reply:
Done!

Reply:
[3288912](https://github.com/autowarefoundation/autoware.universe/pull/1962/commits/3288912a621b137787006ca91fdaa682d1a2d107)